fvt/predictor/predictor_suite_test.go

```diff
@@ -60,8 +62,12 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	// runs *only* on process #1
 	createFVTClient()
 
-	// confirm 3 serving runtimes exist
-	list, err := FVTClientInstance.ListServingRuntimes(metav1.ListOptions{})
+	// confirm 3 cluster serving runtimes or serving runtimes exist
+	f := FVTClientInstance.ListClusterServingRuntimes
+	if NameSpaceScopeMode {
+		f = FVTClientInstance.ListServingRuntimes
+	}
+	list, err := f(metav1.ListOptions{})
```

Comment:
Maybe this is a bit clearer?:
```suggestion
	var err error
	var list *unstructured.UnstructuredList
	if NameSpaceScopeMode {
		list, err = FVTClientInstance.ListServingRuntimes(metav1.ListOptions{})
	} else {
		list, err = FVTClientInstance.ListClusterServingRuntimes(metav1.ListOptions{})
	}
```